packages/tenderly-core/src/internal/core/services/TenderlyService.ts

```diff
@@ -112,19 +135,22 @@ export class TenderlyService {
     username: string
   ): Promise<void> {
     if (!TenderlyApiService.isAuthenticated()) {
-      console.log(`Error in ${this.pluginName}: ${ACCESS_TOKEN_NOT_PROVIDED_ERR_MSG}`);
+      logger.error(`Error in ${this.pluginName}: ${ACCESS_TOKEN_NOT_PROVIDED_ERR_MSG}`);
       return;
     }
 
     const tenderlyApi = TenderlyApiService.configureInstance();
     try {
+      logger.debug("Making a call to the api...");
       const res = await tenderlyApi.post(`/api/v1/account/${username}/project/${tenderlyProject}/contracts`, {
         ...request,
       });
+      logger.debug("Call to the api successfully made.");
+      logger.trace("Retrieved data:", res.data);
```

Comment:
be careful here with logging the entire response data. This might have some sensitive information.

Reply:
@dule-git can you elaborate why you resolved this comment and decided to let the `res.data` being logged as is? (Just trying to understand the reasoning...)